src/Builders/EmailBuilder.cs

```diff
@@ -19,18 +19,35 @@ private static MimeMessage BuildMessage(EmailMessage emailMessage)
             var toEmails = emailMessage.ToEmail.Split(new string[] { "," }, StringSplitOptions.RemoveEmptyEntries);
 
             foreach (var email in toEmails)
-            {
                 message.To.Add(new MailboxAddress(string.Empty, email.Trim()));
-            }
 
             message.From.Add(new MailboxAddress(string.Empty, emailMessage.FromEmail));
 
             if (!string.IsNullOrEmpty(emailMessage.CcEmail))
                 message.Cc.Add(new MailboxAddress(string.Empty, emailMessage.CcEmail));
+            
+            var htmlBody = new TextPart("html");
+            htmlBody.Text = emailMessage.Body;
+            Multipart multipart = new("mixed");
+            multipart.Add(htmlBody);
```

Comment:
This could be constructed more succinctly, by using curlies on the new object being created.